packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.tsx

```diff
@@ -206,6 +230,15 @@ export default function CreateTableViewWizard(
     sql,
     viewTypeMask,
   ])
+
+  const onTableSql = useCallback(async () => {
+    try {
+      await validateSql()
+    } catch (e) {
+      return
+    }
```

Comment:
It's not a good idea to swallow errors, though in this case we know that the error is shown to the user after it is returned by `useMutation`.

To remove this bad pattern, let's use `mutate` instead of `mutateAsync`. `mutate` will not throw on an error, so there is no need to `try/catch`.

I think then `onTableSql` could be deleted (since it would just call another fn) and in `onNextButtonClicked` you can just call `validateSql`

Reply:
May or may not want to keep this fn after updating the mutation hook per [this comment](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/811/files#r1523625082) as you will still need to call `validateSql({sql, entityType})`